lib/spatial_features/importers/ogr.rb

```diff
@@ -0,0 +1,25 @@
+require 'ostruct'
+require 'digest/md5'
+require 'spatial_features/importers/geo_json'
+
+module SpatialFeatures
+  module Importers
+    class OGR < GeoJSON
```

Comment:
Should we call this something to do with ESRI instead of just `OGR`? It seems like is it parsing ESRI JSON?

Reply:
I'll leave piping for later. Renamed as suggested.